### ISSUE TITLE
Wrong Testing result from demo code

### ISSUE THREAD

**Issue by Inosonnia** [kampta/DeepLayout#4]
Thanks for your repo.

I used the default training code of publayout, and trained for 10 epoches.
The final training loss reaches around 1.27.
epoch 10 iter 5245: train loss 1.27828. lr 2.880000e-04: 100%|██████████| 5246/5246 [46:58<00:00,  1.86it/s]

I opened the annotated code groups (total 4), and found that the input image is fine while the output seems not satisfactory.
`                for i, layout in enumerate(layouts):
                    layout = self.test_dataset.render(layout)
                    layout.save(os.path.join(self.config.samples_dir, f'recon_{epoch:02d}_{i:02d}.png'))`

![image](https://user-images.githubusercontent.com/11433902/142362797-6242b926-6ac0-406b-8669-da34f8b9b3a8.png)

Any suggestions or comments will be appreciated. thanks!
